mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg

```diff
@@ -26,6 +26,15 @@ If tradeable = false, then even after you've discovered the item it still won't
 IF THE WaitForDiscovery OPTION ISN'T SHOWING UP, BE SURE TO RUN THE GAME AT LEAST ONCE WITH BetterTrader INSTALLED! THE OPTION WILL BE GENERATED ON STARTUP!
 
 #Better Trader#
+Amber: Stack = 1 Price = 5 SellPrice = 5 Tradeable = true IgnoreWaitForDiscovery = true Sellable = true
```

Comment:
Given everything else, I think I really want to encourage world foraging, so, I would suggest the price of goods such as Amber, Rubies and other valuables (Silver Necklace? I think there are a few more still) go up substantially.  Closer to 10x the current value.  Not sure why they're tradable, but I'm also not opposed to it even at 1:1

Reply:
I don't see any issue with this, I can come up some formulaic way to increase them